backend/Modules/Users/HitMeApp.Users/Application/Handlers/IntegrationEvents/UserRegisteredHandler.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Threading.Tasks;
+using HitMeApp.Identity.IntegrationEvents;
+using HitMeApp.Shared.Infrastructure.Integration;
+using Serilog;
+
+namespace HitMeApp.Users.Application.Handlers.IntegrationEvents
+{
+    internal sealed class UserRegisteredHandler : IIntegrationEventHandler<UserRegistered>
+    {
+        private readonly ILogger _logger;
+
+        public UserRegisteredHandler(ILogger logger)
+        {
+            _logger = logger;
+        }
+
+        public Task Handle(UserRegistered @event)
+        {
+            _logger.Information("Handling {eventName} event with payload: {@event}", nameof(UserRegistered), @event);
+            return Task.CompletedTask;
+        }
```

Comment:
Is this `TODO`? No handling is being made here.

Reply:
There's nothing to handle as the user module is empty now. Just wanted to illustrate how integration should be handled.
I will add TODO to avoid confusion.